CHANGELOG.md

```diff
@@ -8,6 +8,22 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.4] - 2023-02-02
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.3...v0.3.4)
+
+### Fixed
+
+* Fixed the environment variable mapping (#498). For the full list of environment variables see [.config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
+
+### Added
+
+* Added OpenTelemetry tracing (#499)
```

Comment:
Let's add a quick follow up PR to openfga/openfga.dev that documents how to wire up tracing (including the flags and docker compose if that's what you're using).